### ISSUE TITLE
Problems installing the package

### ISSUE THREAD

**Issue by samoncode** [Kensuke-Mitsuzawa/sumo-output-parsers#12]
Hi, 
I am trying to install the the sumo-output-parsers via pip.
But get the following error.

    ERROR: Could not build wheels for cartopy which use PEP 517 and cannot be installed directly

Any tips on how to solve this?

Best regards,
Sam

**Comment by Kensuke-Mitsuzawa** [comment]
Hi. I tested to install it on a Google Colab environment, however, the issue does not raise.

Looking at [the thread](https://github.com/SciTools/cartopy/issues/1239#issuecomment-445520383), the error message is a typical one when your env. does not have specific libraries.

Can you try installing it?
And, if possible, your env. information, please. Which OS, Python version etc.

It's one weird point that sumo-output-parsers installs `cartopy`. I'm sure that I deleted `cartopy` from a dependency package list. Let me double-check.

**Comment by samoncode** [comment]
Thank you for the response,
I can not pin it down at the moment. 
I also tried it on colab just for reference, and unsurprisingly it worked as well. 
But than I tried it on my local machine with different approaches. 
 * Windows 11 - anaconda python 3.7
 * Windows 11 - anaconda python 3.8
 * Windows 11 - venv python 3.7
 * WSL - Ubuntu 18.04 - anaconda python 3.6
 * WSL - Ubuntu 18.04 - anaconda python 3.7
 * WSL - Ubuntu 18.04 - anaconda python 3.8 

Nothing worked, but under windows the issue is different: concerning the fiona installer.

With the Ubuntu settings its always the cartopy installation.

Still working on it.
